script_test.go

```diff
@@ -43,10 +39,36 @@ func (m gobinMain) Run() int {
 }
 
 func TestScripts(t *testing.T) {
+	var (
+		pathToMod     string // local path to this module
+		modTestGOPATH string // GOPATH set when running tests in this module
+	)
+
+	// set pathToMod
+	wd, err := os.Getwd()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to get working directory for test: %v", err)
+	}
+	pathToMod = wd
+
+	// set modTestGOPATH
+	var stdout, stderr bytes.Buffer
+	cmd := exec.Command("go", "env", "GOPATH")
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to get GOPATH for test: %v\n%s", err, stderr.String())
```

Comment:
same  as above

Reply:
Also, I think you could simply use `exec.Command(...).Output()` for this. The error returned by it contains stderr, if I remember correctly.

Reply:
Done